CHANGELOG.md

```diff
@@ -9,34 +9,89 @@ The version headers in this history reflect the versions of Apollo Server itself
 
 ## v3.0.0 (PRERELEASE)
 
-> The changes noted within this `v3.0.0` section have not been finalized.  This section aims to track all changes that are going into the 3.0 release, but we will not claim that it is comprehensive until we have finalized the 3.0 release.
+> The changes in this `v3.0.0` section have not been finalized. This section aims to track all changes that are going into the 3.0 release, but it is not comprehensive until the 3.0 release is finalized.
```

Comment:
interesting, you don't like the active voice here?  (also below I'm noticing I have "is being compiled to ES2020" which probably has a better wording.)

Reply:
Not so much active vs. passive as it is including "us" in the language, feels reasonable to keep it more neutrally perspectived, but this was far from a critical edit on my part 🤷‍♂️